python/cugraph/cugraph/sampling/uniform_neighbor_sample.py

```diff
@@ -142,6 +144,15 @@ def uniform_neighbor_sample(
         will return the renumber map and renumber map offsets
         as an additional dataframe.
 
+    retain_seeds: bool, optional (default=False)
+        If True, will retain the original seeds (original source vertices)
+        in the output even if they do not have outgoing neighbors.
+
+    label_offsets: integer sequence, optional (default=None)
+        Offsets of each label within the start vertex list.
+        Only used if retain_seeds is True.  Required if retain_seeds
+        is True.
```

Comment:
Just a suggestion: maybe change this so that if this is specified, the `retain_seeds=True` behavior is assumed and then drop the `retain_seeds` arg completely?  If you decide to do that, then be sure to also include the description of the `retain_seeds` behavior in this description.

Reply:
That's a reasonable suggestion.  I'll add that to the new PR.